src/screens/AddSiteScreen.js

```diff
@@ -1,204 +1,339 @@
 import React, {Component} from 'react';
```

Comment:
A lot of things look like they changed below, because my auto-formatter moved a bunch of stuff. I'll try to comment on all of the actual relevant changes.